__tests__/app.test.js

```diff
@@ -134,3 +134,28 @@ describe("GET /api/articles/:article_id/comments", () => {
             });
     });
 });
+
+describe("POST /api/articles/:article_id/commments", () => {
+    test("201 returns comment posted after inserting to given article", () => {
+        const newComment = {
+            username: "butter_bridge",
+            body: "test body",
+        };
+        return request(app)
+            .post("/api/articles/3/comments")
+            .send(newComment)
+            .expect(201)
+            .then(({ body }) => {
+                const { comment } = body;
+                expect(typeof comment.comment_id).toBe("number");
+                expect(typeof comment.votes).toBe("number");
+                expect(typeof comment.created_at).toBe("string");
+                expect(typeof comment.author).toBe("string");
+                expect(typeof comment.body).toBe("string");
+                expect(typeof comment.article_id).toBe("number");
+                expect(comment.author).toEqual("butter_bridge");
+                expect(comment.body).toEqual("test body");
+                expect(comment.article_id).toEqual(3);
+            });
```

Comment:
To test the properties of an entire response object, you might find it helpful to use a flexible matcher like [.toMatchObject](https://jestjs.io/docs/expect#expectobjectcontainingobject) or [objectContaining](https://jestjs.io/docs/expect#expectobjectcontainingobject) combined with `expect.any(*data type*)` in order to check this response object - the syntax is a bit nicer, and you'd only need one expect statement to check the whole object - have a look into this!